Can't compile on Windows

Thanks for reporting a bug! Please include all of the following:



```
error: linking with `link.exe` failed: exit code: 1120
  |
  = note: "C:\\Program Files\\Microsoft Visual Studio\\2022\\Enterprise\\VC\\Tools\\MSVC\\14.31.31103\\bin\\HostX64\\x64\\link.exe" "/NOLOGO" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.0.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.1.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.10.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.11.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.12.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.13.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.14.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.15.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.2.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.3.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.4.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.5.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.6.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.7.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.8.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.difft.2229026d-cgu.9.rcgu.o" "C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.4ucpje1yqf4wo9eh.rcgu.o" "/LIBPATH:C:\\CarTar\\release\\deps" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\difftastic-d0bc0fc173dc38be\\out" "/LIBPATH:C:\\CarTar\\release\\build\\libmimalloc-sys-20c3a23102c37094\\out" "/LIBPATH:C:\\CarTar\\release\\build\\tree-sitter-54819667e45c4a8b\\out" "/LIBPATH:C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "/WHOLEARCHIVE:tree-sitter-elisp.lib" "/WHOLEARCHIVE:tree-sitter-yaml.lib" "/WHOLEARCHIVE:tree-sitter-gleam.lib" "/WHOLEARCHIVE:tree-sitter-nix.lib" "/WHOLEARCHIVE:tree-sitter-json.lib" "/WHOLEARCHIVE:tree-sitter-java.lib" "/WHOLEARCHIVE:tree-sitter-bash.lib" "/WHOLEARCHIVE:tree-sitter-dart.lib" "/WHOLEARCHIVE:tree-sitter-commonlisp.lib" "/WHOLEARCHIVE:tree-sitter-python.lib" "/WHOLEARCHIVE:tree-sitter-c.lib" "/WHOLEARCHIVE:tree-sitter-elm.lib" "/WHOLEARCHIVE:tree-sitter-tsx.lib" "/WHOLEARCHIVE:tree-sitter-javascript.lib" "/WHOLEARCHIVE:tree-sitter-lua.lib" "/WHOLEARCHIVE:tree-sitter-yaml-cpp.lib" "/WHOLEARCHIVE:tree-sitter-python-cpp.lib" "/WHOLEARCHIVE:tree-sitter-bash-cpp.lib" "/WHOLEARCHIVE:tree-sitter-ocaml-interface.lib" "/WHOLEARCHIVE:tree-sitter-elixir.lib" "/WHOLEARCHIVE:tree-sitter-go.lib" "/WHOLEARCHIVE:tree-sitter-lua-cpp.lib" "/WHOLEARCHIVE:tree-sitter-ocaml.lib" "/WHOLEARCHIVE:tree-sitter-clojure.lib" "/WHOLEARCHIVE:tree-sitter-css.lib" "/WHOLEARCHIVE:tree-sitter-elixir-cpp.lib" "/WHOLEARCHIVE:tree-sitter-cpp.lib" "/WHOLEARCHIVE:tree-sitter-ocaml-interface-cpp.lib" "/WHOLEARCHIVE:tree-sitter-janet-simple.lib" "/WHOLEARCHIVE:tree-sitter-haskell.lib" "/WHOLEARCHIVE:tree-sitter-ocaml-cpp.lib" "/WHOLEARCHIVE:tree-sitter-zig.lib" "/WHOLEARCHIVE:tree-sitter-typescript.lib" "/WHOLEARCHIVE:tree-sitter-cpp-cpp.lib" "/WHOLEARCHIVE:tree-sitter-scala.lib" "/WHOLEARCHIVE:tree-sitter-ruby.lib" "/WHOLEARCHIVE:tree-sitter-rust.lib" "/WHOLEARCHIVE:tree-sitter-c-sharp.lib" "/WHOLEARCHIVE:tree-sitter-php.lib" "/WHOLEARCHIVE:tree-sitter-ruby-cpp.lib" "/WHOLEARCHIVE:tree-sitter-php-cpp.lib" "/WHOLEARCHIVE:tree-sitter-haskell-cpp.lib" "C:\\CarTar\\release\\deps\\libterm_size-a606155b2cf59322.rlib" "C:\\CarTar\\release\\deps\\libwu_diff-25706f2c56b595d3.rlib" "C:\\CarTar\\release\\deps\\libwalkdir-f1e1ee70fbc01ed1.rlib" "C:\\CarTar\\release\\deps\\libsame_file-bc9e1fcdb3a9ce29.rlib" "C:\\CarTar\\release\\deps\\libmimalloc-a34e6c4878df8f00.rlib" "C:\\CarTar\\release\\deps\\liblibmimalloc_sys-5637947aacd9e5d7.rlib" "C:\\CarTar\\release\\deps\\libtree_sitter-218357aeaae69ec2.rlib" "C:\\CarTar\\release\\deps\\libtyped_arena-7014457b2934d18d.rlib" "C:\\CarTar\\release\\deps\\libconst_format-02c2a773f3a8982f.rlib" "C:\\CarTar\\release\\deps\\libclap-181dce83c392cf05.rlib" "C:\\CarTar\\release\\deps\\libtextwrap-15592219c0da512c.rlib" "C:\\CarTar\\release\\deps\\libindexmap-9ba0747720636ee8.rlib" "C:\\CarTar\\release\\deps\\libhashbrown-da79696167b25bce.rlib" "C:\\CarTar\\release\\deps\\libos_str_bytes-866c1ce66d33c507.rlib" "C:\\CarTar\\release\\deps\\libbitflags-fc1965db0469ec95.rlib" "C:\\CarTar\\release\\deps\\libowo_colors-98e60874a82ac78b.rlib" "C:\\CarTar\\release\\deps\\liblazy_static-1ef46232122641ee.rlib" "C:\\CarTar\\release\\deps\\libstrsim-6bb7f88b582d93fe.rlib" "C:\\CarTar\\release\\deps\\librpds-46b113ddf608761e.rlib" "C:\\CarTar\\release\\deps\\libarchery-99a38dcda4cc1b30.rlib" "C:\\CarTar\\release\\deps\\libradix_heap-aeee00374f9c6ec6.rlib" "C:\\CarTar\\release\\deps\\libitertools-f88c3d6a944d7f69.rlib" "C:\\CarTar\\release\\deps\\libeither-e5de8338724a2e66.rlib" "C:\\CarTar\\release\\deps\\librustc_hash-8313ee8ab007b3cf.rlib" "C:\\CarTar\\release\\deps\\libpretty_env_logger-0d1df7f7af835f66.rlib" "C:\\CarTar\\release\\deps\\libenv_logger-0350867492d53208.rlib" "C:\\CarTar\\release\\deps\\libtermcolor-34d3957e6613ab0f.rlib" "C:\\CarTar\\release\\deps\\libwinapi_util-a475941558562532.rlib" "C:\\CarTar\\release\\deps\\libatty-77062c7dfe6818ec.rlib" "C:\\CarTar\\release\\deps\\libwinapi-d5424493765e0873.rlib" "C:\\CarTar\\release\\deps\\libhumantime-ebe0a80c385dc9b7.rlib" "C:\\CarTar\\release\\deps\\libquick_error-6eb9b497be5ca6be.rlib" "C:\\CarTar\\release\\deps\\libregex-c8a7feb7cd5e3697.rlib" "C:\\CarTar\\release\\deps\\libaho_corasick-79387342b0d73800.rlib" "C:\\CarTar\\release\\deps\\libmemchr-76dbfe21baa53138.rlib" "C:\\CarTar\\release\\deps\\libregex_syntax-b2ea96671453d68b.rlib" "C:\\CarTar\\release\\deps\\liblog-533992625f374413.rlib" "C:\\CarTar\\release\\deps\\libcfg_if-d092cf17aa22092c.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libstd-9505d954cc550ca4.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libpanic_unwind-4368ec34df66b968.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_demangle-c4e960482b87deda.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libstd_detect-3e30157661c6e929.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libhashbrown-441bbf33311d8f21.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_std_workspace_alloc-a65adc14aa7ec78a.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libunwind-754bada3b2f76f43.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcfg_if-bbc7454d63bcee48.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liblibc-083a8721e26e7568.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liballoc-fc82f5be4815a343.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_std_workspace_core-14f675cf009ff7ec.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcore-39142ba17f25ae37.rlib" "C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcompiler_builtins-1a5802a39db22341.rlib" "advapi32.lib" "cfgmgr32.lib" "gdi32.lib" "kernel32.lib" "msimg32.lib" "opengl32.lib" "user32.lib" "winspool.lib" "kernel32.lib" "ws2_32.lib" "bcrypt.lib" "advapi32.lib" "userenv.lib" "kernel32.lib" "msvcrt.lib" "/NXCOMPAT" "/LIBPATH:C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "/OUT:C:\\CarTar\\release\\deps\\difft-c8ab8d3c92a269de.exe" "/OPT:REF,ICF" "/DEBUG" "/NATVIS:C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\intrinsic.natvis" "/NATVIS:C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\liballoc.natvis" "/NATVIS:C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libcore.natvis" "/NATVIS:C:\\Users\\dschroeder\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libstd.natvis"
  = note:    Creating library C:\CarTar\release\deps\difft-c8ab8d3c92a269de.lib and object C:\CarTar\release\deps\difft-c8ab8d3c92a269de.exp
          tree-sitter-elm.lib(parser.o) : error LNK2001: unresolved external symbol tree_sitter_elm_external_scanner_create
          tree-sitter-elm.lib(parser.o) : error LNK2001: unresolved external symbol tree_sitter_elm_external_scanner_destroy
          tree-sitter-elm.lib(parser.o) : error LNK2001: unresolved external symbol tree_sitter_elm_external_scanner_scan
          tree-sitter-elm.lib(parser.o) : error LNK2001: unresolved external symbol tree_sitter_elm_external_scanner_serialize
          tree-sitter-elm.lib(parser.o) : error LNK2001: unresolved external symbol tree_sitter_elm_external_scanner_deserialize
          C:\CarTar\release\deps\difft-c8ab8d3c92a269de.exe : fatal error LNK1120: 5 unresolved externals


error: failed to compile `difftastic v0.26.3`, intermediate artifacts can be found at `C:\CarTar`

Caused by:
  could not compile `difftastic` due to previous error
```

I've also encountered this issue with a pretty similar error
```
error: linking with `link.exe` failed: exit code: 1120
  |
  = note: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX64\\x64\\link.exe" "/NOLOGO" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.0.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.1.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.10.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.11.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.12.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.13.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.14.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.15.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.2.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.3.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.4.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.5.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.6.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.7.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.8.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.difft.25834a4f-cgu.9.rcgu.o" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.4ajp6taqxageywop.rcgu.o" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\difftastic-4903dbb3a9628a4a\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\libmimalloc-sys-6f985e420a494d3c\\out" "/LIBPATH:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\build\\tree-sitter-8578933e86ed4b16\\out" "/LIBPATH:C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "/WHOLEARCHIVE:tree-sitter-elisp.lib" "/WHOLEARCHIVE:tree-sitter-json.lib" "/WHOLEARCHIVE:tree-sitter-clojure.lib" "/WHOLEARCHIVE:tree-sitter-gleam.lib" "/WHOLEARCHIVE:tree-sitter-yaml.lib" "/WHOLEARCHIVE:tree-sitter-bash.lib" "/WHOLEARCHIVE:tree-sitter-javascript.lib" "/WHOLEARCHIVE:tree-sitter-nix.lib" "/WHOLEARCHIVE:tree-sitter-lua.lib" "/WHOLEARCHIVE:tree-sitter-go.lib" "/WHOLEARCHIVE:tree-sitter-dart.lib" "/WHOLEARCHIVE:tree-sitter-css.lib" "/WHOLEARCHIVE:tree-sitter-janet-simple.lib" "/WHOLEARCHIVE:tree-sitter-commonlisp.lib" "/WHOLEARCHIVE:tree-sitter-toml.lib" "/WHOLEARCHIVE:tree-sitter-python.lib" "/WHOLEARCHIVE:tree-sitter-lua-cpp.lib" "/WHOLEARCHIVE:tree-sitter-ocaml-interface.lib" "/WHOLEARCHIVE:tree-sitter-yaml-cpp.lib" "/WHOLEARCHIVE:tree-sitter-elixir.lib" "/WHOLEARCHIVE:tree-sitter-bash-cpp.lib" "/WHOLEARCHIVE:tree-sitter-python-cpp.lib" "/WHOLEARCHIVE:tree-sitter-ruby.lib" "/WHOLEARCHIVE:tree-sitter-java.lib" "/WHOLEARCHIVE:tree-sitter-typescript.lib" "/WHOLEARCHIVE:tree-sitter-zig.lib" "/WHOLEARCHIVE:tree-sitter-scala.lib" "/WHOLEARCHIVE:tree-sitter-elm.lib" "/WHOLEARCHIVE:tree-sitter-ocaml-interface-cpp.lib" "/WHOLEARCHIVE:tree-sitter-elixir-cpp.lib" "/WHOLEARCHIVE:tree-sitter-cpp.lib" "/WHOLEARCHIVE:tree-sitter-ocaml.lib" "/WHOLEARCHIVE:tree-sitter-ruby-cpp.lib" "/WHOLEARCHIVE:tree-sitter-rust.lib" "/WHOLEARCHIVE:tree-sitter-elm-cpp.lib" "/WHOLEARCHIVE:tree-sitter-tsx.lib" "/WHOLEARCHIVE:tree-sitter-kotlin.lib" "/WHOLEARCHIVE:tree-sitter-php.lib" "/WHOLEARCHIVE:tree-sitter-cpp-cpp.lib" "/WHOLEARCHIVE:tree-sitter-ocaml-cpp.lib" "/WHOLEARCHIVE:tree-sitter-php-cpp.lib" "/WHOLEARCHIVE:tree-sitter-haskell.lib" "/WHOLEARCHIVE:tree-sitter-c-sharp.lib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libterm_size-7229cefb5dbb73c8.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libwu_diff-6892e0e12a2d9464.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libwalkdir-37064d29a8751267.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libsame_file-136e98a4beb966c2.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libmimalloc-6868667319fb3c72.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\liblibmimalloc_sys-9bd179d0cd305ae8.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libtree_sitter-6590961b2445e7dd.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libtyped_arena-e91d2138091f3909.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libconst_format-37ceb88409288922.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libclap-5f379010f41f1850.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libtextwrap-06f6182ec23d96fe.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libindexmap-494d96940bff1ff6.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libhashbrown-0a7e11755adfc239.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libos_str_bytes-c49f8586d6b8928d.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libbitflags-1028cba9f96399be.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libowo_colors-2e83a1659f3a7396.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\liblazy_static-4e4e10c698982bb3.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libstrsim-3c8088533bba2555.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\librpds-3aa74fa024ba76f6.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libarchery-fde1824e4aefeab4.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libradix_heap-36106c8d39630101.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libitertools-6e30eb78299242f4.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libeither-1a027c9d5f153506.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\librustc_hash-d2cf773396b92299.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libpretty_env_logger-6122d75f46cea611.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libenv_logger-09f6c35b1abc1cf7.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libtermcolor-fbbd9ccb2132b87d.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libwinapi_util-6beeb58fc7e12c9d.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libatty-1773ba577ffff502.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libwinapi-4bdebb036b0dfbf7.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libhumantime-52b21445169789f8.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libquick_error-7a7466fb35650fb8.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libregex-2d627859bd3e7396.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libaho_corasick-26ae88a3e055a149.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libmemchr-1165996c9a421532.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libregex_syntax-c32d36601fdb58e1.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\liblog-a9512dfac16cc07b.rlib" "C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\libcfg_if-e36650d725906808.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libstd-77c29e3b2a96c9a6.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libpanic_unwind-fac20c79897f2b3d.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libstd_detect-a900fec85d21ec5f.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_demangle-2803b5471132ab91.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libhashbrown-bd7c3f8e84ab3746.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_std_workspace_alloc-6353ac840b4a82ca.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libunwind-8b22f250a6b6c0c3.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcfg_if-837411c15bbbd755.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liblibc-6e0180ba426c6f71.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liballoc-8fee164e10a5c1ee.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_std_workspace_core-433995d9d73cd404.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcore-d681750c6d1718a3.rlib" "C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcompiler_builtins-fd343f19f347f62a.rlib" "advapi32.lib" "cfgmgr32.lib" "gdi32.lib" "kernel32.lib" "msimg32.lib" "opengl32.lib" "user32.lib" "winspool.lib" "kernel32.lib" "ws2_32.lib" "bcrypt.lib" "advapi32.lib" "userenv.lib" "kernel32.lib" "msvcrt.lib" "/NXCOMPAT" "/LIBPATH:C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "/OUT:C:\\Users\\Chase\\AppData\\Local\\Temp\\cargo-installxK5FlF\\release\\deps\\difft-4c58f491c2051e70.exe" "/OPT:REF,ICF" "/DEBUG" "/NATVIS:C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\intrinsic.natvis" "/NATVIS:C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\liballoc.natvis" "/NATVIS:C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libcore.natvis" "/NATVIS:C:\\Users\\Chase\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libstd.natvis"
  = note:    Creating library C:\Users\Chase\AppData\Local\Temp\cargo-installxK5FlF\release\deps\difft-4c58f491c2051e70.lib and object C:\Users\Chase\AppData\Local\Temp\cargo-installxK5FlF\release\deps\difft-4c58f491c2051e70.exp
          difft-4c58f491c2051e70.difft.25834a4f-cgu.5.rcgu.o : error LNK2019: unresolved external symbol tree_sitter_c referenced in function _ZN5difft18tree_sitter_parser13from_language17haabc72d59d16da1fE
          C:\Users\Chase\AppData\Local\Temp\cargo-installxK5FlF\release\deps\difft-4c58f491c2051e70.exe : fatal error LNK1120: 1 unresolved externals

error: failed to compile `difftastic v0.27.0`, intermediate artifacts can be found at `C:\Users\Chase\AppData\Local\Temp\cargo-installxK5FlF`
```

also, on windows 11, difftastic v27:
```
          difft-3594e9aa2074f155.difft.6c00d299-cgu.5.rcgu.o : error LNK2019: unresolved external symbol tree_sitter_python referenced in function _ZN5difft18tree_sitter_parser13from_language17h44979f1d781cdb5fE
          difft-3594e9aa2074f155.difft.6c00d299-cgu.5.rcgu.o : error LNK2019: unresolved external symbol tree_sitter_dart referenced in function _ZN5difft18tree_sitter_parser13from_language17h44979f1d781cdb5fE
          tree-sitter-elixir.lib(parser.o) : error LNK2001: unresolved external symbol tree_sitter_elixir_external_scanner_create
          tree-sitter-elixir.lib(parser.o) : error LNK2001: unresolved external symbol tree_sitter_elixir_external_scanner_destroy
          tree-sitter-elixir.lib(parser.o) : error LNK2001: unresolved external symbol tree_sitter_elixir_external_scanner_scan
          tree-sitter-elixir.lib(parser.o) : error LNK2001: unresolved external symbol tree_sitter_elixir_external_scanner_serialize
          tree-sitter-elixir.lib(parser.o) : error LNK2001: unresolved external symbol tree_sitter_elixir_external_scanner_deserialize
          C:\Users\IDOBAR~1\AppData\Local\Temp\cargo-installtm1EWR\release\deps\difft-3594e9aa2074f155.exe : fatal error LNK1120: 7 unresolved externals


error: failed to compile `difftastic v0.27.0`, intermediate artifacts can be found at `C:\Users\IDOBAR~1\AppData\Local\Temp\cargo-installtm1EWR`
```

--- a/build.rs
+++ b/build.rs
@@ -22,13 +22,6 @@ impl TreeSitterParser {
             }
         }
 
-        let mut build = cc::Build::new();
-        build.include(&dir).warnings(false); // ignore unused parameter warnings
-        for file in c_files {
-            build.file(dir.join(file));
-        }
-        build.compile(self.name);
-
         if !cpp_files.is_empty() {
             let mut cpp_build = cc::Build::new();
             cpp_build
@@ -58,6 +51,13 @@ impl TreeSitterParser {
             }
             cpp_build.compile(&format!("{}-cpp", self.name));
         }
+
+        let mut build = cc::Build::new();
+        build.include(&dir).warnings(false); // ignore unused parameter warnings
+        for file in c_files {
+            build.file(dir.join(file));
+        }
+        build.compile(self.name);
     }
 }
 

--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.56"
+channel = "1.57"
 components = ["rustfmt"]
 profile = "minimal"

